src/components/layout/contextFilters/contextFilters.js

```diff
@@ -40,8 +41,8 @@ class ContextFilters extends Component {
     return (
       <div className="context-filter-container">
         <div className="device-group-filter">
-          <img src={FilterSvg} alt="Filter" className="filter-icon" />
-          <div className="select-container">
+          <img src={FilterSvg} alt="Filter" className={disableDeviceFilter ? "filter-icon disabled" : "filter-icon"} />
+          <div className={disableDeviceFilter ? "select-container disabled" : "select-container"}>
```

Comment:
always show `select-container`, the condition should decide wether or not to show disabled. Same comment for the filter-icon